### PR TITLE
@pepopowitz => [SearchResults] Add pagination to artworks tab

### PR DESCRIPTION
**Pull request by mzikherman**
search

Adds in our pagination component - this works seamlessly with our searchable connection structure. Other tabs should follow.

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 494f805e7c37a3ecd597a564796b7deb813afe07

https://deploy-preview-2150--artsy-reaction.netlify.com

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit bd68a1f30ef5c8dc12572d9d555338faf803fe7a

https://deploy-preview-2150--artsy-reaction.netlify.com

**Comment by mzikherman** [comment]
Added spinner (plus scrolling to the top- we can always remove this behavior if we don't like it).

search

**Comment by mzikherman** [comment]
Lol that gif totally doesn't show it- the API requests are too fast for the gif frame rate! Here's one with an artificial delay:

search

File: src/Components/v2/index.tsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ export * from "./CountrySelect"
 export * from "./FollowIcon"
 export * from "./Footer"
 export * from "./Lightbox"
+export * from "./LoadingArea"
```
Comment (mzikherman) [inline src/Components/v2/index.tsx]: There were quite a few places (including other apps), reaching into the artist app (which will make refactoring and moving apps around harder, later). So I just made that its own component out of the apps directory.

File: src/Apps/Artist/ArtistApp.tsx [hunk 2]
```diff
@@ -6,12 +6,12 @@ import { HorizontalPadding } from "Apps/Components/HorizontalPadding"
 import { track } from "Artsy/Analytics"
 import * as Schema from "Artsy/Analytics/Schema"
 import { PreloadLinkState } from "Artsy/Router/Components/PreloadLink"
+import { LoadingArea } from "Components/v2/LoadingArea"
```
Comment (alloy) [inline src/Apps/Artist/ArtistApp.tsx]: What does `Components/v2` mean?

Comment (mzikherman) [reply src/Apps/Artist/ArtistApp.tsx]: cc @damassi this is our convention, all new components go there (vs old `Styleguide`). I'm just following the convention! Chris will be able to explain reasoning more.

**Comment by artsyit** [comment]
:rocket: PR was released in v15.3.0 :rocket:

**Pull request merged**